administration.html

```diff
@@ -0,0 +1,272 @@
+        <div class="row">
+            <div class="column ">
+                <h1>Pas de file d'attente dans ma mairie</h1>
+                <p>À chaque rentrée vos usagers doivent faire la queue pour faire
+                    calculer leurs aides, certains posent même une demi-journée pour faire ces démarches.</p>
+                <p>La bonne nouvelle ?
+                    <a style="color: #1631d5;" href="https://particulier.api.gouv.fr/FAQ.html">API Particulier résout ce
+                        problème</a>.
+                    <b> API Particulier est un service gratuit fourni par la <abbr
+                            title="Direction interministérielle du numérique et du système d'information et de communication de l'État">DINSIC</abbr>.</b>
+                </p>
+                <div>
+                    <form id="gform"
+                          action="https://script.google.com/macros/s/AKfycbzSuEezvl9QSpGbtOsP_F__g_A-vua8KgWrIK9K/exec"
+                          method="post" style="margin: 0">
+                        <div id="form-elements">
+                            <div style="margin-top: 0.5em">
+                                <input id="gform-input-name" type="text" placeholder="Mon administration (ex : Maire de Clamart)" name="name">
+                            </div>
+                            <div style="margin-top: 0.5em">
+                                <input type="email" placeholder="Mon adresse email (ex : claude@ville-clamart.fr)"
+                                       name="email">
+                            </div>
+                            <div class="honeypot">
+                                <input type="text" name="honeypot">
+                            </div>
+                            <div style="margin-top: 1em">
+                                <button class="button" type="submit">Faire un test pour mon administration</button>
+                            </div>
+                        </div>
+                        <div id="thank-you-message" class="notification success" style="display: none">Merci ! Nous
+                            revenons
+                            vers vous au plus vite.
+                        </div>
+                    </form>
+                </div>
+            </div>
+            <div class="column illustration">
+                <img src="img/file-d-attente.png" alt="File d'attente">
+            </div>
+        </div>
+    </div>
+</div>
+
+<section class="">
+    <div class="container">
+        <h2 class="section__title"><a href="#">37 administrations</a> ont déjà opté pour API Particulier avec des
+            résultats comme :</h2>
+        <div class="row">
+            <div class="card">
+                <div class="card__content">
+                    <p style="font-size: 1.3em; font-style: italic; text-align: center">
+                        "En 2014, nos usagers posaient une demi-journée pour effectuer leur démarche en mairie.
```

Comment:
I liked the statistic/quote format.
Is there any way to put that in ?
Suggestions:
*Des démarches dématérialisées de bout en bout*
"En 2014, nos usagers posaient une demi-journée pour effectuer leur démarche en mairie. Aujourd'hui, ils le font en ligne en 5 minutes."
- Simon Pierre, DGA action à la population, Clamart

*80% des usagers font les démarches entièrement en ligne*
"Maintenant que le traitement est automatisé, j'ai enfin le temps d'aider les usagers en difficulté, dans certains cas je peux les contacter de manière proactive."
- Aurélie, directrices affaires générales, Clamart

*500.000 pièces justificatives évités en septembre 2018*
"API Particulier, c'est génial, il n'y a plus rien à scanner !"
- Aurélie, directrices affaires générales, Clamart

Reply:
:+1: